ansible/configs/ibm-rhoic/infra.yml

```diff
@@ -245,7 +245,7 @@
           [?name=='vpc-{{ guid }}']
       register: r_update_sg
       tags: update-sg-tcp
-      until: r_update_sg.status != 500
+      until: r_update_sg.status == 201
```

Comment:
This should be 200

Reply:
Fixed, also gave each of the two updates different variables for the return just in case